vllm/worker/model_runner.py

```diff
@@ -1003,16 +1021,30 @@ def capture_model(self, kv_caches: List[torch.Tensor]) -> None:
                     graph_runner.flashinfer_decode_wrapper = \
                         decode_wrapper
 
-                graph_runner.capture(
+                capture_inputs = {
+                    "input_ids":
                     input_tokens[:batch_size],
+                    "positions":
                     input_positions[:batch_size],
+                    "hidden_states":
                     hidden_states[:batch_size]
                     if hidden_states is not None else None,
+                    "kv_caches":
                     kv_caches,
+                    "attn_metadata":
                     attn_metadata,
-                    memory_pool=self.graph_memory_pool,
-                    stream=graph_capture_context.stream,
-                )
+                    "memory_pool":
+                    self.graph_memory_pool,
+                    "stream":
+                    graph_capture_context.stream
+                }
+                if self.has_seqlen_agnostic:
```

Comment:
nit: can we add a comment explaining why this is here? e.g. it is only used by mamba model cuda graph

Reply:
Added a comment